vizier/shared/src/info/vizierdb/artifacts/VegaChart.scala

```diff
@@ -1663,9 +1667,18 @@ object VegaValueReference
     target: VegaValueReference
   ) extends VegaValueReference
 
+  /**
+   * Used for Width of bar chart 
+   */
+  case class ScaleBandRef(
+    scale: String,
+    band: Option[Int]
+    )extends VegaValueReference
```

Comment:
consistent formatting please.  No leading spaces and a space between `)` and `extends`